Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -356,3 +357,148 @@ void parseCXExtensions(RDKit::RWMol &mol, const std::string &extText,
   }
 }
 }  // end of namespace SmilesParseOps
+namespace RDKit {
+namespace SmilesWrite {
+namespace {
+std::string quote_string(const std::string &txt) {
```

Comment:
It looks like this isn't being used or perhaps is a place holder.

Reply:
yeah, it's a place holder. Technically output strings need to be quoted if they contain non-standard characters. I haven't written that yet.

Reply:
But I should at least be calling it from the appropriate places. I've updated the PR to do so.